### PR TITLE
docs: add pact-cypress-adapter

### PR DESCRIPTION
**Pull request by B3nnyL**
Pact Cypress Adapter is a collection of cypress commands that generate pact from Cypress test.

### CODE REVIEW

**Comment by mefellows** [comment]
Hi team - just following up on this PR. Do you need anything else from us to progress it?

**Comment by B3nnyL** [comment]
@elylucas is there anything else required from us?

**Review by admah — APPROVED** [review]
Looks like a great addition. Thanks!

File: content/_data/plugins.json [hunk 1]
```diff
@@ -643,6 +643,14 @@
           "link": "https://picklejs.com",
           "keywords": ["cucumber", "collection", "actions", "commands"],
           "badge": "community"
+        }, 
+        {
+          "name": "Pact Cypress Adapter",
+          "description": "Simple commands that generates pact from Cypress for pact test",
+          "link": "https://github.com/pactflow/pact-cypress-adapter",
+          "keywords": ["pact", "contract testing", "commands"],
+          "badge": "official"
```
Comment (admah) [inline content/_data/plugins.json:652]:
```suggestion
          "badge": "community"
```

**Review by admah — CHANGES_REQUESTED** [review]
Actually I noticed one thing that needs to change. This needs to be listed as a community plugin.

File: content/_data/plugins.json (hunk 1 above)
Comment (YOU54F) [reply content/_data/plugins.json:652]: Good shout @admah 

We've also address a description change we wanted to make, in the following PR https://github.com/cypress-io/cypress-documentation/pull/4477

@B3nnyL is based in Aus, so will probably be 💤  :)

Comment (YOU54F) [reply content/_data/plugins.json:652]: Happy for her to update the comments and get the full PR glory though 🙌🏾  🚀

**Comment by B3nnyL** [comment]
@YOU54F thanks! I'll close this PR due to duplication of #4477

**Pull request closed without merging**